src/commonComponents/paragraph/Paragraph.js

```diff
@@ -0,0 +1,35 @@
+// Libraries
+import PropTypes from 'prop-types'
+
+// classNames
+import cx from 'classnames'
+
+// lodash
+import _noop from 'lodash/noop'
+
+
+function Paragraph(props){
+
+    const { label, isActive, handleClick } = props;
+    const classNames = cx({
```

Comment:
may be you can call this **paragraphClass** or **paraClass** or **paragraphClassNames**, right now it's fine there's only single element inside this component but if you would have more elements then naming should be specific